### PR TITLE
OADP-2719: Release notes for OADP 1.2.2

### PR DESCRIPTION
**Pull request by anarnold97**
* [OADP-2719](https://issues.redhat.com/browse/OADP-2719): Release Notes 1.2.2 OADP.

* Enterprise 4.11 → branch/enterprise-4.11
* Enterprise 4.12 → branch/enterprise-4.12
* Enterprise 4.13 → branch/enterprise-4.13
* Enterprise 4.14 → branch/enterprise-4.14

* For more details about the information requested in this template, see:
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/create_or_edit_content.adoc#submit-PR --->

Version(s):

Issue:

https://issues.redhat.com/browse/OADP-2719

## Link to docs preview:

* [OADP 1.2.2 release notes](https://65604--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-release-notes#migration-oadp-release-notes-1-2-2_oadp-release-notes)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/oadp-release-notes-1-2-2.adoc
Comment (amastbau) [inline modules/oadp-release-notes-1-2-2.adoc:53]: I think we should change the description here (or even the issue title)
This issue might/probably not related to kubevirt fs-overhead
even if it is, Users might bumped into this issue outside of kubevirt.
@shubham-pampattiwar  wdyt?

File: modules/oadp-release-notes-1-2-2.adoc
Comment (amastbau) [inline modules/oadp-release-notes-1-2-2.adoc:84]: Jira issue link is missing

**Review by amastbau — CHANGES_REQUESTED** [review]

File: modules/oadp-release-notes-1-2-2.adoc
Comment (shubham-pampattiwar) [reply modules/oadp-release-notes-1-2-2.adoc:53]: @amastbau yes I agree with you, we can add something more generic like, (due to storage provider implementation choices) whenever there is a difference between the application PVC storage request and the snapshot size of the same pvc this issue will occur and OADP 1.2 datamover is able to handle that now after this fix.

**Comment by anarnold97** [comment]
@amastbau && @shubham-pampattiwar - made the requested, please can i get a LGTM if you are happy? GA: General Availability | 10/16/2023
Thanks

**Review by shubham-pampattiwar — APPROVED** [review]

**Comment by anarnold97** [comment]
@shubham-pampattiwar - thanks very much

File: modules/oadp-release-notes-1-2-2.adoc
Comment (skrthomas) [inline modules/oadp-release-notes-1-2-2.adoc:24]:
```suggestion
.Restic restore partially failed due to a Pod Security standard
```

File: modules/oadp-release-notes-1-2-2.adoc
Comment (skrthomas) [inline modules/oadp-release-notes-1-2-2.adoc:21]:
```suggestion
The following highlighted issues are resolved in OADP 1.2.2:
```

Comment (skrthomas) [reply modules/oadp-release-notes-1-2-2.adoc:21]: i think you can just say "are" here since resolved is past tense already.

File: modules/oadp-release-notes-1-2-2.adoc
Comment (skrthomas) [inline modules/oadp-release-notes-1-2-2.adoc:28]: Is it upgrade to either one of these releases? Or do they need to upgrade to 1.1.6 and then 1.2.2?

File: modules/oadp-release-notes-1-2-2.adoc
Comment (skrthomas) [inline modules/oadp-release-notes-1-2-2.adoc:81]:
```suggestion
The `oc adm must-gather` command fails to remove `ClusterRoleBinding` resources, which are left on cluster due to admission webhook. Therefore, requests for the removal of the `ClusterRoleBinding` resources are denied. link:https://issues.redhat.com/browse/OADP-2773[(OADP-27730)]
```

File: modules/oadp-release-notes-1-2-2.adoc
Comment (skrthomas) [inline modules/oadp-release-notes-1-2-2.adoc:35]:
```suggestion
In previous releases of OADP 1.2, the backup of an application with internal images partially failed with plugin panicked error returned. The backup partially fails with this error in the Velero logs:
```

File: modules/oadp-release-notes-1-2-2.adoc
Comment (skrthomas) [inline modules/oadp-release-notes-1-2-2.adoc:61]:
```suggestion
.Possible pod volume backup failure if Velero was installed in multiple namespaces
```

Comment (skrthomas) [reply modules/oadp-release-notes-1-2-2.adoc:61]: Looking in the repo, I see Velero capitalized, unless its in code/error message

File: modules/oadp-release-notes-1-2-2.adoc
Comment (skrthomas) [inline modules/oadp-release-notes-1-2-2.adoc:63]:
```suggestion
In previous releases of OADP 1.2, there was a possibility of pod volume backup failure if Velero was installed in multiple namespaces. link:https://issues.redhat.com/browse/OADP-2409[(OADP-2409)]
```

File: modules/oadp-release-notes-1-2-2.adoc
Comment (skrthomas) [reply modules/oadp-release-notes-1-2-2.adoc:81]: backticks needed for the resource name and the inline code.

**Review by skrthomas — COMMENTED** [review]
A few comments for you :) Nice work!

**Comment by anarnold97** [comment]
@skrthomas - thanks. changes being made and squashed

File: modules/oadp-release-notes-1-2-2.adoc
Comment (skrthomas) [inline modules/oadp-release-notes-1-2-2.adoc:53]:
```suggestion
In previous releases of OADP 1.2, due to storage provider implementation choices, whenever there was a difference between the application persistent volume claims (PVCs) storage request and the snapshot size of the same PVC, VM's using filesystemOverhead failed when backing up and restoring. This issue has been resolved in the Data Mover of OADP 1.2.2. link:https://issues.redhat.com/browse/OADP-2144[(OADP-2144)]
```

Comment (skrthomas) [reply modules/oadp-release-notes-1-2-2.adoc:53]: Oop need to remove this space

File: modules/oadp-release-notes-1-2-2.adoc
Comment (anarnold97) [reply modules/oadp-release-notes-1-2-2.adoc:28]: It has been resolved in 1.1.6 and also 1.2.2

**Pull request merged**
